examples/Example projects/LedStripDriver_OpenSmart/LedStripDriver_OpenSmart.ino

```diff
@@ -0,0 +1,165 @@
+/*
+   Homey Arduino library
+   Usage example for use with Wemos D1 and Led Strip Driver (open-smart)
+
+   Homey library: https://github.com/athombv/homey-arduino-library
+   Driver for P9813 chip: ChainableRGB: https://github.com/pjpmarques/ChainableLED
+
+   Hardware:
+   - WeMos D1 mini V2
+   - RGB LED Strip Driver Module Shield
+   
+*/
+
+#include <ESP8266WiFi.h>        //ESP8266 library
+#include <WiFiClient.h>         //ESP8266 library
+#include <Homey.h>              //Athom Homey library
+#include <ChainableLED.h>       //P9813 driver library
+
+//just for debugging to serial.print: use DEBUG_PRINT instead of Serial.print
+//exclude Serial.print (x) for non debug like: #define DEBUG_PRINT(x) //Serial.piint(x)
+ #define DEBUG_PRINT(x)     Serial.print (x)
+ #define DEBUG_PRINTLN(x)   Serial.println (x)
+
+
+//configuration LED driver
+#define NUM_LEDS  1   //number of leds (chips) in series
+#define LED_PIN_C D3  //clock pin
+#define LED_PIN_D D4  //data pin
+boolean ledOnOff = false;
+float ledHue = 0;
+float ledSaturation = 0;
+float ledBrightness = 0;
+
+//define relais
+#define RELAIS_PIN D2 //activate relais
```

Comment:
(not really an issue but a single RELAY or multiple RELAIS?)

Reply:
Nou het is 1 relais, meervoud is ook ook relais maar in dit geval gaat het om 1 stuk.